modules/javafx.controls/src/main/java/javafx/scene/control/skin/MenuButtonSkinBase.java

```diff
@@ -185,7 +192,10 @@ public MenuButtonSkinBase(final C control) {
                 // through the mnemonics code (especially in case they should be
                 // consumed to prevent them being used elsewhere).
                 // See JBS-8090026 for more detail.
-                Platform.runLater(() -> Utils.removeMnemonics(popup, getSkinnable().getScene()));
+                Scene scene = getSkinnable().getScene();
```

Comment:
`Utils.addMnemonics` has the same check, so in case of macOS `mnemonics`  list is always empty

Reply:
I was wondering why this code works.
Apparently, Utils.addMnemonics() adds to 'mnemonics' list only with same condition if (!com.sun.javafx.PlatformUtil.isMac()) { ...
Hence,  mnemonics list is empty in case platform is macOs.